src/views/Home.vue

```diff
@@ -87,60 +83,87 @@ export default {
 }
 
 #intro {
+  display: grid;
```

Comment:
whoa! I've never seen the grid display type before. I will have to check this out.